numpy/lib/_arraysetops_impl.py

```diff
@@ -412,14 +412,10 @@ def _unique_all_dispatcher(x, /):
 @array_function_dispatch(_unique_all_dispatcher)
 def unique_all(x):
     """
-    Find the unique elements of an array, and counts, inverse and indices.
+    Find the unique elements of an array, and counts, inverse, and indices.
 
-    This function is an Array API compatible alternative to:
-
-    >>> x = np.array([1, 1, 2])
-    >>> np.unique(x, return_index=True, return_inverse=True,
-    ...           return_counts=True, equal_nan=False)
-    (array([1, 2]), array([0, 2]), array([0, 0, 1]), array([2, 1]))
+    This function is an Array API compatible alternative to `np.unique`, but
+    returns a namedtuple for easier access to each output.
```

Comment:
I think this removes too much info - the idea here is that the `unique_*` functions behave like `unique` with the given subset of keyword arguments changed. Those changes include `equal_nan`, which differs than the default value of `np.unique` as well.

I'd propose to leave the info here, but remove the execution bit:

```rst
    This function is an Array API compatible alternative to::
    
        np.unique(x, return_index=True, return_inverse=True,
                  return_counts=True, equal_nan=False)

    but returns a namedtuple for easier access to each output.
```